docs/_source/practical_guides/create_dataset.md

```diff
@@ -572,50 +642,15 @@ for record in dataset.records:
 dataset.update_records(modified_records)
 ```
 
-###### Update `suggestions`
-
-You can also add suggestions to existing records that have been already pushed to Argilla:
-
-```python
-import argilla as rg
-
-rg.init(api_url="<ARGILLA_API_URL>", api_key="<ARGILLA_API_KEY>")
-
-dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
-```
-
-::::{tab-set}
-
-:::{tab-item} Argilla 1.14.0 or higher
-
-```python
-for record in dataset.records:
-    record.update(suggestions=[{"question_name": "question", "value": ...}]) # Directly pushes the update to Argilla
+```{note}
+As soon as you update the records in a remote dataset, the changes should be available in the Argila UI. If you cannot see them, try hitting the `Refresh` button on the sidebar.
 ```
 
-:::
-
-:::{tab-item} Lower than Argilla 1.14.0
-
-```python
-for record in dataset.records:
-    record.set_suggestions([{"question_name": "question", "value": ...}])
-dataset.push_to_argilla() # No need to provide `name` and `workspace` as has been retrieved via `from_argilla` classmethod
+```{note}
+Only fields cannot be added, modified or removed from the records, as this would compromise the consistency of the dataset.
```

Comment:
`external_id` cannot be updated neither. Just `metadata` , `vectors`, `suggestions` and  `responses` should be "updatables" cc @gabrielmbmb

Reply:
Is it that `external_id` cannot be modified once it's there or that it cannot be added/modified/deleted at all once the record is pushed to argilla?

Reply:
I believe that is correct.